src/scss/components/optional/_menu.scss

```diff
@@ -18,6 +18,7 @@ $menu-border-width                      : 1px;
 $menu-border-width--active              : 2px;
 $menu-link-color                        : $color-text;
 $menu-link-padding                      : spacing() spacing(x2);
+$menu-positionTop                       : $stickyHeader-height + spacing(x2);
```

Comment:
Just pulled in the latest version of fozzie and I'm getting errors around this variable:

`Undefined variable: "$stickyHeader-height".`

Does this get defined anywhere atm?

Reply:
It's defined in Menu Web which would explain why we didn't see the error when testing, sorry! The value is always 57px. Do you want a fix PR?

Reply:
Yes please 👍